__tests__/app.test.js

```diff
@@ -34,3 +35,23 @@ describe("GET/api/topics", () => {
       });
   });
 });
+
+describe("GET/api", () => {
+  test("status: 200, should respond with a json file describing each endpoint", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(endpoints);
+      });
+  });
+
+  test("404: responds with error message page not found", () => {
```

Comment:
You can do a test for a 404, but it shouldn't be in the same describe block for this ticket as it's not specific to this endpoint. Please separate this out 👍